src/systems/joint_controller/JointController.cc

```diff
@@ -142,7 +142,16 @@ void JointController::Configure(const Entity &_entity,
   }
   if (_sdf->HasElement("topic"))
   {
-    topic = _sdf->Get<std::string>("topic");
+    topic = transport::TopicUtils::AsValidTopic(
+        _sdf->Get<std::string>("topic"));
+
+    if (topic.empty())
+    {
+      ignerr << "Failed to create topic for joint ["
+             << _sdf->Get<std::string>("topic")
+             << "]" << std::endl;
```

Comment:
The error message looks wrong
```suggestion
      ignerr << "Failed to create topic [" << this->dataPtr->jointName
             << "]" << for joint [" << _sdf->Get<std::string>("topic")
             << "]" << std::endl;
```